flink-python/pyflink/gen_protos.py

```diff
@@ -120,10 +120,17 @@ def _install_grpcio_tools_and_generate_proto_files():
     logging.warning('Installing grpcio-tools into %s', install_path)
     try:
         start = time.time()
-        subprocess.check_call(
-            [sys.executable, '-m', 'pip', 'install',
-             '--prefix', install_path, '--build', build_path,
-             '--upgrade', GRPC_TOOLS, "-I"])
+        pip_version = pkg_resources.get_distribution("pip").version
```

Comment:
Add some comment here that why we need this if else.

Reply:
Done